KsApi/models/User.swift

```diff
@@ -12,7 +12,7 @@ public struct User {
   public let name: String
   public let newsletters: NewsletterSubscriptions
   public let notifications: Notifications
-  public let optedOutOfRecommendations: Bool
+  public let optedOutOfRecommendations: Bool?
```

Comment:
What's the benefit of making this an optional instead of a default value?

Reply:
We have an `UserAttribute` enum that returns `Lens<User, Bool?>` and that's because notifications and newsletter subscriptions lenses are of type  `Lens<User, Bool?>`. optedOutOfRecommendations is another user property so it needs to conform to this type. Happy to chat in person if this doesn't make any sense.